docs/cryptography/contributionSigning.md

```diff
@@ -0,0 +1,126 @@
+# Contribution Signing
+
+In order to verify that contributions listed in the `transcript.json` are actually generated by the addresses listed in the transcript, the contributors MAY sign their contribution with their Ethereum address that they logged in with. And in turn, the secret contributed to the ceremony can be used as a BLS private key which can then sign the participant's Ethereum address or their GitHub name.
```

Comment:
Ack, this `MAY` is important because it means that you do not need to do the double signing procedure. If you want to build your participant code just like how you would build it for a regular ceremony then it is still possible

Reply:
Yup, definitely an optional feature. Debated MAY vs SHOULD, is this OPTIONAL or RECOMMENDED?